RecoParticleFlow/PFClusterProducer/python/particleFlowClusterECAL_cff.py

```diff
@@ -4,6 +4,10 @@
 particleFlowClusterECAL.energyCorrector.applyMVACorrections = True
 particleFlowClusterECAL.energyCorrector.maxPtForMVAEvaluation = 90.
 
+from Configuration.Eras.Modifier_run2_ECAL_2016_cff import run2_ECAL_2016
+run2_ECAL_2016.toModify(particleFlowClusterECAL,
+                        energyCorrector = dict(srfAwareCorrection = True, maxPtForMVAEvaluation = 300.))
+
 from Configuration.Eras.Modifier_run2_ECAL_2017_cff import run2_ECAL_2017
 run2_ECAL_2017.toModify(particleFlowClusterECAL,
                         energyCorrector = dict(srfAwareCorrection = True, maxPtForMVAEvaluation = 300.))
```

Comment:
```(run2_ECAL_2016 | run2_ECAL_2017).toModify``` is more compact and also reasserts that the values are the same

Reply:
> I was looking at the differences in the new modifiers compared to the existing 2017 ones
> 
>     * run2_ECAL_2017 also applies changes in hcalIsoTrackStudy_cff.py and HcalIsoTrkAnalyzer_cff.py
> 
> 
> Is the run2_ECAL_2016 applicable there as well?

I am not so sure - this code is for HCAL studies, so probably someone from them can answer? or from ECAL (Amina or Fabrice)?

Reply:
> I am not so sure - this code is for HCAL studies, so probably someone from them can answer? or from ECAL (Amina or Fabrice)?

Indeed, this is not an e/gamma problem and is not relevant to this PR other than noting that there is a potential issue to follow up with the appropriate people.